ui/hooks/useMultichainAccountTotalFiatBalance.test.tsx

```diff
@@ -15,31 +15,21 @@ const mockTokenBalances = [
     balance: '48573',
     balanceError: null,
     decimals: 6,
-    image: undefined,
-    isERC721: undefined,
-    string: '0.04857',
+    string: 0.04857,
     symbol: 'USDC',
+    tokenFiatAmount: '0.05',
   },
   {
     address: '0x0bc529c00C6401aEF6D220BE8C6Ea1667F6Ad93e',
     symbol: 'YFI',
     balance: '1409247882142934',
     decimals: 18,
-    string: '0.001409247882142934',
+    string: 0.00141,
```

Comment:
This looks like the values are changing, but it actually keeps them (mostly) the same.  The old behavior of mocking `./useTokenTracker` was modifying this array during the test, so this just updates to the same final result. 

<img width="523" alt="Screenshot 2024-11-15 at 11 22 22 AM" src="https://github.com/user-attachments/assets/92ee8004-bef3-4af6-a489-cfa22a5a8c66">